lib/tasks/curator_tasks.rake

```diff
@@ -43,27 +43,15 @@ namespace :curator do
       raise 'app:db:migrate and db:migrate rake tasks are not available!'
     end
 
-    if Rake::Task.task_defined?('app:db:test:prepare')
```

Comment:
If we remove this step (`app:db:test:prepare`), when I run the commands below from the project root dir:
```
$ rails curator:setup
$ bundle exec rake spec
```
I get an error:
```
Migrations are pending. To resolve this issue, run:
        rails db:migrate RAILS_ENV=test
```
I added this block back in, ran `rake db:drop`, then re-ran the commands above and everything worked.

Reply:
Added it back in but slightly different. That original task I was invoking is said to be deprecated in the future which is why I ended up removing it. Also added bundle exec before a couple system calls to it to make it safer in the environment its being run in.